packages/quick_actions/quick_actions/android/src/main/java/io/flutter/plugins/quickactions/QuickActionsPlugin.java

```diff
@@ -43,6 +47,10 @@ public void onDetachedFromEngine(FlutterPluginBinding binding) {
   @Override
   public void onAttachedToActivity(ActivityPluginBinding binding) {
     handler.setActivity(binding.getActivity());
+    binding.addOnNewIntentListener(this);
+    if (isInitialized) {
```

Comment:
Why is this (and the new method) needed? I don't see any deferment logic, so couldn't it just send the call unconditionally?

Reply:
Ah my bad. Initially I thought it'll call onNewIntent when the Dart side isn't initialized yet, which means it'll make a redundant method call which would just vanish. But after checking again, the method call does go through to Dart.

Removed the if block, and the new method. Thanks for catching this.